main.go

```diff
@@ -51,5 +53,11 @@ func main() {
 		PhotoRouter:          photoRouter,
 		PhotoDir:             photoDir,
 	}
-	server.Init(config)
+
+	if err := server.Init(config); err != nil {
```

Comment:
It's hard to know that server.init actually includes running the server only when trace into it.

Reply:
fix it in #25